home/urls.py

```diff
@@ -2,8 +2,10 @@
 from . import views
 
 app_name = 'home'
+
 urlpatterns = [
-    path('', views.index, name='index'),
+    path('new/', views.index, name='new_index'),
+    path('', views.old_index, name='index'),
```

Comment:
Old page should use `views.index` while new page should use `views.new_index`.

Reply:
Ohh Sorry, that's my bad....I completely forgot that